packages/ui/src/pages/Home/MultisigView.tsx

```diff
@@ -134,6 +141,12 @@ const ListFieldText = styled('div')`
   color: ${({ theme }) => theme.custom.gray[800]};
 `
 
+const ListFieldValue = styled(Typography)`
+  color: ${({ theme }) => theme.custom.gray[800]};
```

Comment:
The other values don't have the same color, it's `theme.custom.text.secondary`
```suggestion
  color: ${({ theme }) => theme.custom.text.secondary};
```